includes/library/zencart/listingBox/src/boxes/LeadCountries.php

```diff
@@ -6,7 +6,7 @@
  * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
  * @version GIT: $Id:$
  */
-namespace ZenCart\Platform\listingBox\boxes;
+namespace ZenCart\ListingBox\boxes;
 
 /**
  * Class Index
```

Comment:
If the \Zencart\Admin  namespace should be gone, then the various docblocks should also be updated ... ie:

```php
/*
 * @package ZenCart\Admin\Controllers
 */
```

And composer.json should probably remove this as well:

```
     "ZenCart\\Admin\\": "admin/includes/library/zencart/"
```